client/src/pages/Help/Help.js

```diff
@@ -0,0 +1,283 @@
+import React from "react";
+import {
+  Grid,
+  Link,
+  Typography,
+  Breadcrumbs,
+  Container,
+  Avatar,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  responsiveFontSizes,
+} from "@material-ui/core";
+import { Link as RRDLink, useRouteMatch } from "react-router-dom";
+import Nav from "../../components/Nav";
+import teamLogo from "../../pages/Help/images/teamLogo.png";
+import homePage from "../../pages/Help/images/homePage.png";
+import adminPage from "../../pages/Help/images/adminPage.png";
+import helpPage from "../../pages/Help/images/helpPage.png";
+
+const Help = () => {
+  const imageStyle = {
+    width: "50%",
+    height: "50%",
+  };
+  const accordionHeader = {
+    marginLeft: "20px",
+    fontSize: "20px",
+  };
+  return (
+    <React.Fragment>
+      <Nav />
+
+      <Grid
+        style={{
+          marginTop: "60px",
+          marginLeft: "10px",
+        }}
+      >
+        <Breadcrumbs aria-label="breadcrumb">
+          <Link style={{ cursor: "pointer" }} component="span">
+            <RRDLink
+              to="/"
+              style={{ textDecoration: "none", color: "inherit" }}
+            >
+              Home
+            </RRDLink>
+          </Link>
+          <Link style={{ cursor: "pointer" }} component="span">
+            <RRDLink
+              to="/game"
+              style={{ textDecoration: "none", color: "inherit" }}
+            >
+              Game
+            </RRDLink>
+          </Link>
+          <Link style={{ cursor: "pointer" }} component="span">
+            <RRDLink
+              to="/admin"
+              style={{ textDecoration: "none", color: "inherit" }}
+            >
+              Admin
+            </RRDLink>
+          </Link>
+
+          <Link
+            style={{ cursor: "pointer" }}
+            color="textPrimary"
+            aria-current="page"
+          >
+            <RRDLink
+              to="/help"
+              style={{ textDecoration: "none", color: "inherit" }}
+            >
+              Help
+            </RRDLink>
+          </Link>
+        </Breadcrumbs>
```

Comment:
I think the breadcrumbs should only be 'Home / Help'

Reply:
The user does not navigate through Game / Admin to get to the help page

Reply:
I agree. Home / Help should be the path. Help should also not be a link, since you're already on that page.

Reply:
Ok, will update